deploy/overlays/autopush/ai.yaml

```diff
@@ -0,0 +1,4 @@
+us-central1:
```

Comment:
I don't see model deployed in datcom-website-autopush?

Reply:
Changed with an empty file for now.